life_server/Functions/Housing/fn_houseCleanup.sqf

```diff
@@ -5,15 +5,18 @@
     Description:
     Cleans up containers inside in house of player.
 */
-private ["_query","_containers"];
-_query = format ["SELECT pid, pos FROM containers WHERE pid='%1' AND owned='1'",_this];
+private _query = format ["SELECT pid, pos FROM containers WHERE pid='%1' AND owned='1'",_this];
```

Comment:
I'll mark this to change, as this is quite ambiguous to read without following the execution path (in reference to _this)